src/styles.css/stars.css

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+  margin-bottom: 0;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0),
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
```

Comment:
The `nth-child` selectors are incorrectly set. For example, `.stars--0 .stars__star:nth-child(-n + 0)` will not select any elements because `-n + 0` results in an empty set. You should adjust these selectors to correctly target the intended number of stars. For instance, `.stars--0 .stars__star:nth-child(-n + 1)` should be used to select the first star in `.stars--0`, and so on.